e2e-tests/src/test/emergency_finalizer.rs

```diff
@@ -24,3 +34,117 @@ async fn set_emergency_finalizer_test() -> anyhow::Result<()> {
     assert_eq!(current_finalizer, Some(public));
     Ok(())
 }
+
+async fn finalize_from_to<C: AsConnection + Sync>(
+    connection: &C,
+    from: BlockHash,
+    to: BlockHash,
+    finalizer: AlephKeyPair,
+) -> anyhow::Result<()> {
+    let from = connection.get_block_number(from).await?.unwrap();
+    let to = connection.get_block_number(to).await?.unwrap();
+
+    for i in (from + 1)..to {
```

Comment:
THis won't finalize `to` - is that intended?

Reply:
fixed